commands/user

```diff
@@ -67,7 +67,7 @@ function cmd_create {
 
   local priv
   priv=$(role_to_group "${role}")
- 
+  log_audit "Create user ${name}, ${role}"
```

Comment:
I busctl fails, the user won't be created but there will be an entry in the audit log.
I think that we need to log only successful commands or better yet log the command execution status afterwards.
This comment relates to all commands, not just this one.